frontend/src/app/shared/components/confirmation-dialog.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, Inject } from '@angular/core';
+import { ThemePalette } from '@angular/material/core';
+import { MAT_DIALOG_DATA, MatDialog } from '@angular/material/dialog';
+
+interface ConfirmationDialogData {
+  title?: string;
+  contentText?: string;
+  primaryColor?: ThemePalette;
+}
+
+@Component({
+  selector: 'eko-confirmation-dialog',
+  template: `
+    <h2 mat-dialog-title>{{ title }}</h2>
+    <mat-dialog-content class="mat-typography"
+      >{{ contentText }}
+      <ng-content></ng-content>
+    </mat-dialog-content>
+    <mat-dialog-actions align="end">
+      <button
```

Comment:
For me having confirm/yes action on the right feels much more natural for the web https://ux.stackexchange.com/a/63239/42792 (OK on the left is more of Windows convention)

Reply:
Hmm, I really haven't been thinking about this topic much. That is defenitely an interesting thread. I am using a Windows a lot lately and it has subconscious influence as it seems. Interestingly, when I wrote the first version of confirmation dialog for delete, I was not using Windows at that time, I left my job as a contractor at Google just a few months back and I placed the confirmation button ON THE RIGHT side (see the deleted dialog code) . So it is basically Google vs Microsoft fight, right (just kidding)? Let's choose Google then :)

Reply:
Yeah, like scroll direction 😉  But I think right side confirm is more popular on the internet. Even Microsoft->Github is using right side on the web. Or is it switched on Windows?
![image](https://user-images.githubusercontent.com/119451/82608270-a0aaec80-9bba-11ea-9bee-13ad65e429e1.png)

Reply:
![image](https://user-images.githubusercontent.com/15725585/82608993-f59b3280-9bbb-11ea-89f5-441a8ca3fc25.png)

Reply:
Weird, I don't have them separated but comment is still on the right

Reply:
@chodorowicz  Chrome alert is on the left, thoguh
![image](https://user-images.githubusercontent.com/15725585/82609054-12376a80-9bbc-11ea-8890-59851f2d1c90.png)

Reply:
yeah, the warning is a system browser/system notice